.nvmrc

```diff
@@ -0,0 +1 @@
+6
```

Comment:
This probably works all the way back to `node@0.10`, this is mostly intended for development. Can remove if there are objections.

Looking forward to getting more tooling, specifically testing, in place so we can begin refactoring the core library itself.